### ISSUE TITLE
Chinese Arms Exports Over the Last Two Decades

### ISSUE THREAD

**Issue by tsp2123** [jsoma/data-studio-projects#213]
Please complete all of the following sections, or the ghost of Joseph Pulitzer will spookily dance around your issue! A completed version of this template can be found at https://github.com/jsoma/data-studio-projects/issues/1

# Pitch

In 2013, One Belt One Road defined a doctrine for a Chinese foreign policy future outlining infrastructure and other developmental projects that would set China as a global superpower greatly expanding its sphere of influence. What is less talked about is the expansion of the Chinese Military Industrial complex—and while the United States and other European powers still dominate the international arms trade—examining China's arms transfers may provide a worthy insight into the country's foreign policy goals.  

## Summary

Using data from SIPRI, I would like to examine Chinese arms exports over the last two decades.

I would like to ask the following questions:

To what countries does China export the most arms?

What kind of arms are exported?

Have these countries changed over time or have they stayed relatively constant?

It would be worth comparing these numbers to similar data for the United States.

## Details

**Possible headline(s):** 

**Data set(s):** 
Data can be found in SIPRIs arms transfer database:

https://www.sipri.org/databases/armstransfers

**Code repository:** 

https://github.com/tsp2123/data-studio/tree/master/Project_3

**Possible problems/fears/questions:** 

The initial cleaning of data was definitely a big hurdle and if I want to tackle a few more datasets to make country comparisons, this would need to be tackled again. This was a problem I encountered in my first project and didn't really succeed but I need to find a way to make comparative graphs between two different datasets if this is even possible. (Or I guess in this instance I could concat a second data set and add a column with X country and Y country.  

## Work so far

Here are a few simple Matplotlib graphs. The idea is to further examine the country's with the most arms deliveries from China. Ideally I would like to create a time series with these data-points. But here's some preliminary analysis checking out the top 5 countries and the top military equipment. 

![screen shot 2018-07-24 at 10 55 10 am](https://user-images.githubusercontent.com/39416100/43147508-327068b8-8f31-11e8-9df4-cb1ae74c8133.png)

![screen shot 2018-07-24 at 10 53 04 am](https://user-images.githubusercontent.com/39416100/43147510-339d4558-8f31-11e8-8be9-b0ff8f8a88aa.png)

## Checklist

This checklist must be completed before you submit your draft.

- [x] I have already spent time with my data set, opening it, exploring it, etc
- [ ] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [x] My issue links to my data set(s)
- [x] My issue links to my code repository
- [x] My issue explains what I'd like to explore in the data set
- [ ] My issue includes images - either inspiration or what I've done so far

**Comment by jsoma** [comment]
Please link your dataset, code repository, and what you're looking to find in the dataset. "examining China's arms transfers may provide a worthy insight into the country's foreign policy goals." is too vague to really be a pitch, we want specific questions you can answer.

**Comment by tsp2123** [comment]
Sorry! Left a bunch of tks in there intending to fill it earlier this week. Pitch is filled in now.

**Comment by kevinlitman-navarro** [comment]
- For the first bar chart, I'm not sure what SAM or APC are, so I would find a different way of labeling those so the average reader knows what you are talking about 

- I don't know what unit I'm looking at in the country chart, definitely need a label there

- I think you're right, adding a time dimension is crucial for us to see what's going on here. Especially if you combined with some U.S. data, we could see how two ostensible political adversaries have furthered conflicting agendas with arms exports

**Comment by nickospi** [comment]
I think a heat map would compliment this project (I know Soma says no maps, but..). if you could find coords on the countries you could do sth that looks like a heat map that also looks 'military enough'. Also a legend explaining or showing an outline of these arms would add a lot. I think I would also like to know if these countries have been recently engaged to any armed conflicts.

**Comment by tsp2123** [comment]
# Update
	
## Your project content: images/words/etc
	
So I would have updated earlier but taking SIPRI data and turning it from .rtf to a usable .csv has taken up much of the first half of this project so I'm still in an exploratory phase and my datasets will have to go through a series of transformations before the final product in order to say anything really meaningful about the subject matter.

Here's some prelim graphs of the Top Countries for Chinese and US Arms exports.

[US Exports Update.pdf](https://github.com/jsoma/data-studio-projects/files/2238357/US.Exports.Update.pdf)

[China Exports.pdf](https://github.com/jsoma/data-studio-projects/files/2238358/China.Exports.pdf)

Note the scales of these two graphs. I think it's quite revealing about the subject matter. US military exports still dwarf China, while this isn't represented in my graphs right now. I'm going to have to find a way to do this in my next iterations. 
  
## Any changes in direction or topic?

I'm still in an exploratory phase but there are some changes to my dataset. The other day there was an article in the NYTimes about a satelite station built by the Chinese military in Argentina—the story concluded it was an indication of rising Chinese sphere of influence in America's backyard. This is kinda the story I was trying to tell on a larger level. But I think there are two competing stories here. 

The first, as my graphs above suggests, while the American military establishment is concerned about Chinese military influence increasing in certain regions—overall—American military exports are still several magnitudes higher than Chinese. 

However, what would be interesting to see is a regional level analysis to see "(whether) how Chinese arms exports have changed in certain regions and how these compare to the US."

The other competing story is to look at China independently of the comparison to the United States which I think would be revealing of the foreign policy goals of the country at large. 
  

	
## Problems/Questions

Besides cleaning my data—I'll update this with problems as they arise. Minor issues of how to transform datasets keep coming up but that's about it. 
	
## Checklist
	
- [x] I have included my visuals
- [x] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by mattrehbein** [comment]
I agree with your last comment about how there are so many ways to can take this really compelling topic. I like the first angle you mention better: if you have data showing that Chinese arms companies have recently starting selling to countries that are also customers of US arms dealers, that'd be very interesting to see visualized, both when it started and how it has grown.

Another, simpler approach would be to show how and when China has moved up the ranks of the world's biggest arms exporters. From a quick Google search, it looks like China ranks in the top five or so, but how long has it been there? Can we see a point in time when arms exports increased exponentially? To your point about still not competing with US in quantity, a simple bar chart of the top arms exporting countries for the most recent year should reflect that nicely.

Nice job digging through a huge topic! Identifying the story you want to tell under these circumstances is something I find really really difficult.

**Comment by mattrehbein** [comment]
Also -- I noticed some of your preliminary graphs are cutoff in the pdfs. I had the same issue, and the wonderful Marie helped me fix it by adding this after my ax = .plot code:

plt.tight_layout()

**Comment by pasiegrist** [comment]
Hi there, 

I totally agree with Matt: I f you have data showing that Chinese arms companies have recently starting selling to countries that are also customers of US arms dealers this would be a really nice angle. So maybe you should focus on these countries and show, how their imports from the countries have changed? Could  you show what the US and what the Chinese exported there?

Keep on digging! :)

**Comment by maxarvid** [comment]
Always timely. A statistic often thrown around in my own country is that we end up on the list when measuring arms trade per capita. Something to consider, though given that you are looking at two super powers, maybe force projection as a whole is more interesting. 

If you do feel an urge to include more graphs, I wonder if you could single out one type of exports, such as surface-to-air missiles (given recent years use against a commercial airliner) or another technology that summarizes the industry as a whole (fighter jets also come to mind, but that be just me missing Saab cars).

Great project, I might steal it and do something similar on Sweden/Scandinavia for next week 😭.

**Comment by tsp2123** [comment]
# Update
	
## Your project content: images/words/etc

I've been a little slow with my analysis but here's a graph I'm putting up for some preliminary critique if anyone is interested in telling me about what they think about the colors and styles. I've also identified a few interesting points in this graph which will require some form of explanation. Not necessarily as graphical annotations, perhaps in the text

![china_usa_exports_tv](https://user-images.githubusercontent.com/39416100/43555997-81ec1194-95cb-11e8-8079-795e9056d994.png)

  
## Any changes in direction or topic?
  
Basically, I'm identifying interesting regions to look at. The graph above is SEA. I might do a few graphs that are country specific and a few that are by region. 
	
## Problems/Questions
  
Anyone know how to make a timeseries map on QGIS?

	
## Checklist
	
- [ ] I have included my visuals
- [ ] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [ ] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by sarahslo** [comment]
Style-wise, I'd suggest you use a different font for the labeling. Make it a bit bigger than the labels on the charts. On the charts, all of the measures on the left can be a bold font. What does TIV stand for?

When you make your annotation you don't want it all to line up with each other, you want it close to the data so our eyes don't jump around. i think you can lose one, because you will start to clutter up your chart.

**Comment by tsp2123** [comment]
# Final

## Project visuals/text
Hey Friends! Here are my final graphs. I will label these here but the html file, I used the template to title them in html itself so pardon if its a little unclear here. TIV definition is also given in the html because it's a little long, but in brief, it's basically how SIPRI measures the transfer arms by adjusting for various factors like type of weapons, their age, etc. 

TOP 5 GLOBAL ARMS EXPORTERS 
![top_5_global_b_size_size_b](https://user-images.githubusercontent.com/39416100/43740178-8052e1e8-9998-11e8-919e-473f28462a0d.png)

This story is about China, so here I zoom into the map above. 

![bottom_3_world_size_b](https://user-images.githubusercontent.com/39416100/43740272-c8618962-9998-11e8-9185-105f0bd23036.png)

I wanna know who China, France and Germany export to. Here's the top 10 list. Key carries through
CHINA
![china_top_10_size_b](https://user-images.githubusercontent.com/39416100/43740336-f228920e-9998-11e8-95e8-b90489121464.png)
FRANCE
![france_top_10_size_b](https://user-images.githubusercontent.com/39416100/43740337-f2368b0c-9998-11e8-88ac-9a6a609121e9.png)
GERMANY
![germany_top_10_size_b](https://user-images.githubusercontent.com/39416100/43740338-f2477f52-9998-11e8-9267-1ac58244d3ec.png)

I wanna look more into what China is exporting to its top 3 importers
BANGALDESH
![bangladesh_size_a_2](https://user-images.githubusercontent.com/39416100/43740372-0c187b2a-9999-11e8-80a0-f5ffa8b69a1b.png)
MYANMAR
![myanmar_size_a_1](https://user-images.githubusercontent.com/39416100/43740373-0c22422c-9999-11e8-9ed1-892fe53f4965.png)
PAKISTAN
![pakistan_size_a_0](https://user-images.githubusercontent.com/39416100/43740374-0c2d5658-9999-11e8-901c-90f20be5f057.png)

There's a graf in my story about the type of weapons China exports looking at what the country most exports. That graf compares China's top exports to the United States's top exports. I made a graf to contextualize these. The problem is, in the template, they are side by side, which I feel may lead the reader to see them as having the same scales. But they don't. This becomes apparent from reading but isn't immediately clear by the placing of the graphs. But really all I wanted these graphs to do is to show the enormous gap between units of the same weapon exported by the US and those exported by China where these weapons account for the majority of each country's exports. I used the wording of a Harper's Index to do this. 

![us_china_antitank_size_a_0](https://user-images.githubusercontent.com/39416100/43740568-9763fdc6-9999-11e8-8ed4-4e69b585905f.png)
![us_china_sam_size_a_0](https://user-images.githubusercontent.com/39416100/43740569-9772572c-9999-11e8-8b16-073a13ca3f0b.png)

Let me know what you think



## Details

**Headline:** 
China's Arms Exports Are Still Nowhere Near the United States

**Published website version:**
https://tsp2123.github.io/projects.io/Project_3/html_project/index.html
**Code repository:**
https://github.com/tsp2123/data-studio/tree/master/Project_3
**Final data set(s):** 
https://www.sipri.org/databases/armstransfers
## What did you find to be the most difficult part of this project?
The cleaning for this project was supremely difficult and unfortunately I spent the majority of time cleaning a dataset I really only used for two graphs. I also was unable to make a map in time although this was heavily requested. I definitely plan on it though.
## Are you satisfied with what you produced? Is there anything you would like to change or improve?
I think adding a map would benefit this project as it is such a geographically-centered story. The bar graphs are boring but I couldn't think of a more economical way of telling this story. But if ya'll have ideas for more interesting visualizations to use hmu in your comments. 

Also another problem I'm hoping someone call help solve: the template keeps messing up the formatting of my body text. Not sure why this is happening but it's really screwing with the page : ( 
## Checklist

- [x] I have included my visuals
- [x] I have posted my project to my project website
- [x] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by sarahslo** [comment]
thank you for keeping the scales the same!

i think your first two charts could be fever lines - in cases like this, it's the form to use - when you want to compare the trends over time. The colors you use in this chart tho, are not strong enough to separate from the background or from each other. They are all very muted. can you make them stronger they separate better?

the charts with 2 data points; those can be sentences. 2 data points don't need to be charts.

the exports by weapon - stories there for sure. good analysis.

**Comment by tsp2123** [comment]
Thanks for the comments Sarah! My intention for muting the colors for everything but China in the first two graphs was because the story was about China and I didn't want the higher bars to distract from that but I still wanted them there to put perspective. 

But it's good to know I could have used fever lines for these graphs, I was a little nervous about that because I think we got advice from Bui that we should only use line graphs when charting rates.
